api/swagger.yaml

```
@@ -173,6 +173,20 @@ tags:
     x-displayName: "System"
 
 definitions:
+  Capabilities:
+    description: "Represents engine capabilities"
+    type: "object"
+    properties:
+      v_:
+        description: "The version of capabilities this response represents."
+        type: integer
+        x-nullable: false
```

Also see other comment; it's good to put example values per-field, or at least it helps prevent diverging from the actual definition (so easier to maintain);


```suggestion
        type: integer
        x-nullable: false
        example: 1
```